bundle/manifests/opentelemetry-operator.clusterserviceversion.yaml

```diff
@@ -204,6 +204,9 @@ spec:
               - args:
                 - --metrics-addr=127.0.0.1:8080
                 - --enable-leader-election
+
+                command:
```

Comment:
This has been recently removed. I believe the CI should/would fail as this isn't in sync with the source.

Reply:
ok i go fix